NSS-Day-Cohort-42/nutshell-lawn-artists
# Delete button renders when user clicks Edit. Articles are deleted when user clicks delete.

### Changes Made

1. Modified `ArticleProvider`, `ArticleList`, and `ArticleCard`.

### Steps to Review

```
git fetch --all
git checkout mp-delete-article
```
- [ ] From Terminal > Navigate to `api` folder in repo > run `json-server -p 8088 -w database.json`
- [ ] Open new Terminal window >  Navigate to `src` in repo > Run `serve` > (⌘) + 'double-click' on Local URL.
- [ ] Test app functionality:
    - Click on `Edit` to edit one of the articles.
    - Make changes and save them
    - Expectation: Changes are now reflected in the Article list.
    - Click on `Edit` to edit another one of the articles.
    - Click on the `Delete Article` button.
    - Expectation: Article is removed from the list of Articles and from the API
- [ ] View code > Confirm file modifications.
